php/Terminus/User.php

```diff
@@ -73,6 +75,12 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
+  public function instruments() {
```

Comment:
I'd rather just make the `instruments` property public and initialize it in the User initializer. That way we we have a persistent association rather creating a new instruments collection everytime this accessor is called (though that might just be a defect rather than your intent).